specs/category.spec.ts

```diff
@@ -0,0 +1,63 @@
+describe('InstantSearch - Search on specific category', () => {
+  it('navigates to the e-commerce demo', async () => {
+    await browser.url('examples/e-commerce/');
+  });
+
+  it('selects "Appliances" category in list', async () => {
+    await browser.setSelectedHierarchicalMenuItem('Appliances');
+  });
+
+  it('selects "Small Kitchen Appliances" category in list', async () => {
+    await browser.setSelectedHierarchicalMenuItem('Small Kitchen Appliances');
+  });
+
+  it('must have the expected results for "Small Kitchen Appliances"', async () => {
+    const hitsTitles = await browser.getHitsTitles();
```

Comment:
we should assert that the hits "category" (which only shows one level in this example) is "Appliances" for every hit

Reply:
I don't really see what value it would add. Since we're already asserting on the hits title any problem during the refinement will already make the test to fail.